src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -714,6 +774,19 @@ def onZoomOutClicked(self):
             setText("{:.2f}".format(self.dicom_view.zoom * 100) + "%")
         self.draw_roi_window_viewport_zoom_input.setCursorPosition(0)
 
+    def zoom_slider_value_changed(self):
```

Comment:
The zoom slider also needs to change when the top zoom in/out toolbar buttons are pressed. You could probably do this using a signal and slot, otherwise, check how the slices have done it.

This could be a problem as It would call this method, you could always pass an extra boolean to determine whether to carry out the updated logic.